docs/source/advanced_features/storage_config.rst

```diff
@@ -1,49 +1,38 @@
-###############
- Cloud Storage
-###############
+################
+ Storage Config
+################
 
-| All data on TensorBay are hosted on cloud.
-| TensorBay supports two cloud storage modes:
+TensorBay supports two storage config modes:
 
-    - DEFAULT CLOUD STORAGE: data are stored on TensorBay cloud
-    - AUTHORIZED CLOUD STORAGE: data are stored on other providers' cloud
+    - GRAVITI STORAGE CONFIG: storage provided by graviti.
+    - AUTHORIZED STORAGE CONFIG: storage provided by userself.
 
-***********************
- Default Cloud Storage
-***********************
-| In default cloud storage mode, data are stored on TensorBay cloud.
-| Create a dataset with default storage:
+************************
+ GRAVITI Storage Config
+************************
+In graviti storage mode, data are stored to graviti storage space on TensorBay.
 
-.. literalinclude:: ../../../docs/code/getting_started_with_tensorbay.py
-      :language: python
-      :start-after: """Create a Dataset"""
-      :end-before: """"""
+***************************
+ Authorized Storage Config
+***************************
 
-**************************
- Authorized Cloud Storage
-**************************
+When using authorized storage config, dataset are stored on user's storage space and are only indexed to the TensorBay.
+See `authorized storage instruction <https://docs.graviti.cn/guide/tensorbay/data/authorize>`_ for details about how to configured authorized storage on TensorBay.
 
-| You can also upload data to your public cloud storage space.
-| Now TensorBay support following cloud providers:
+Tensorbay also support to authorize cloud or local storage.
```

Comment:
`Tensorbay supports both authorize cloud  storage and authorize local storage.` will be better?